misc/richpresence.txt

```diff
@@ -1,11 +1,11 @@
-"lang"
-{
-	"english"
-    {
-	    "tokens"
-	    {
-	    	"#InGame"	"%levelname%: %time%s"
-	    	"#InMenu"	"In menu"
-	    }
-    }
-}
+"lang"
+{
+	"english"
+    {
+	    "tokens"
+	    {
+	    	"#InGame"	"%levelname% [x%difficultymult%] - %time%s"

```

Comment:
There is also an issue where the time takes a long time for it to get updated on Steam. This always seems to update when the player dies, which makes me believe it's not updating because it's constantly sending requests to the Steam API every frame.

Reply:
A card has been raised for this—#227.